Gemfile

```diff
@@ -1,3 +1,5 @@
+source "https://rubygems.org"
```

Comment:
can we remove the `source 'https://rubygems.org' do` below? Seems duplicative to have both

Reply:
yeah we should be able to remove the `do/end` block below. [Example](https://github.com/dfed/swift-async-queue/blob/a232ddbfb776a3d3e380dce30e7d5f617cffcf82/Gemfile#L3-L5).